python/mxnet/cython/ndarray.pyx

```diff
@@ -13,7 +13,7 @@ cdef class NDArrayBase:
     cdef int cwritable
 
     cdef _set_handle(self, handle):
-        cdef unsigned long ptr
+        cdef unsigned long long ptr
```

Comment:
msvc long is 32bit.long long is 64bit.
in msvc,gcc,clang, long long all is 64bit.